wandb/integration/keras/magic_impl.py

```diff
@@ -0,0 +1,571 @@
+from __future__ import absolute_import
```

Comment:
@raubitsj may have some thoughts on where to put this.  I don't think it belongs inside of `keras`, the magic implementation was designed to support non-keras usecases as well.  Maybe `integration/magic`?

Reply:
integration/magic sounds good for now.

Reply:
On doing this refactor, (`integration/keras/magic_impl.py` ->`integration/magic.py`)  the tfkreras patching for getting validation data fails. Looking into it now. Probably something to do with import order.

Reply:
Fixed.